web/chat/chat-tabs.css

```diff
@@ -0,0 +1,43 @@
+div.container {
+  position: absolute;
+  width: 400px;
+  top: 0;
+  bottom: 0;
+  overflow-y: auto;
+  background-color: white;
+  border-right: 2px solid #E4E4E4;
+}
+
+.tabList {
+  border-bottom: 1px solid #ccc;
+  padding-left: 0;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+}
+  
+.tabItem {
```

Comment:
When defining a CSS selector, let's always specify the element type (unless we need to use the selector for multiple element types): `button.tabItem`, `button.tabItemActive`